webui/react/src/components/HumanReadableFloat.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+import { CommonProps } from 'types';
+
+interface Props extends CommonProps {
+  num: number;
+  precision?: number;
+}
+
+const defaultProps: Props = {
+  num: 0,
+  precision: 6,
+};
+
+const HumanReadableFloat: React.FC<Props> = ({ num, precision }: Props) => {
+  let numToString: string = num.toFixed(precision);
+  if (num < 0.01 || num > 999) {
+    numToString = num.toExponential(precision);
+  }
+  return <span title={num.toString()}>{numToString}</span>;
```

Comment:
**suggestion:** Use `antd` component `Tooltip` instead of the `span` to allow the user to hover over the component to see the original unfiltered number.